4/Dockerfile

```diff
@@ -17,7 +17,7 @@ RUN set -ex \
   ; do \
     gpg --keyserver hkp://pgp.mit.edu:80 --recv-keys "$key" || \
     gpg --keyserver hkp://keyserver.pgp.com:80 --recv-keys "$key" || \
-    gpg --keyserver hkp://ha.pool.sks-keyservers.net:80 --recv-keys "$key" ; \
+    gpg --keyserver hkp://ipv4.pool.sks-keyservers.net:80 --recv-keys "$key" ; \
```

Comment:
Using `ipv4.pool...` does _not_ guarantee the server you are hitting will respond appropriately to requests on port 80 (that's what the `p80` pool is for).

Reply:
So is there a point to using `:80` for `pool.sks-keyservers.net`?